packages/components/src/Combobox/Combobox.types.ts

```diff
@@ -145,6 +145,7 @@ export interface ComboboxOptionProps {
   readonly customRender?: (
     option: Omit<ComboboxOptionProps, "customRender"> & {
       isSelected: boolean;
+      DefaultContent: () => ReactElement;
```

Comment:
undecided on the syntax & name

`defaultContent` vs `DefaultContent`

`defaultComponent` vs `DefaultComponent`

Reply:
sounds good, we'll go with `defaultContent`

reason for the type was because we do want it to be more restrictive. `ReactNode` has things like string, but we know this is an actual component/element

I'll see about changing the type but initially that was intentional since we know it's never going to be most of the possible types of ReactNode